src/i18n/en-US/index.ts

```diff
@@ -1002,6 +1002,11 @@ export default {
       text2: 'Available on Astar zkEVM. Switch the network to use it.',
       learn: '(learn more)',
     },
+    zkatanaBridge: {
```

Comment:
zKatana

Reply:
fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/1238/commits/73807824eb5f82f09b41027309290152d704db13) commit